### DISCUSSION TITLE
Delete a data point

### DISCUSSION THREAD

**Discussion by nonix** [ytyou/ticktock#78] [Q&A]
Hi is there a way to delete some data from tsdb? I did try to do:
`$CURL -XPOST "http://$HOST:$PORT/api/admin?cmd=delete --start 2024-11-29 18:00 --stop 2024-11-29 18:01"`
It did not failed, but did not deleted the data. Am I missing something?

**Comment by nonix** [comment]
Reason: At one point "18:00" there was a sensor mulfunction and temp collected was about -17° (which is a way wrong) this is disrupting the stats.

**Reply by ylin30** [reply]
TT doesn't have API to support delete data explicitly. We should have failed the API. Nevertheless you can do two things to delete data:
1. You can override data points with new values. TT will take the latest values instead of averaging them. 
2. Delete whole TSDB folders (1 TSDB per day). That's probably the last thing you want to do.

**Reply by ylin30** [reply]
Be careful that you must insert the same data point with the same timestamp in the same time series. Otherwise it will be considered as a new data point.
